packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -459,41 +454,23 @@ export function isNetwork(chainId: ChainId) {
   const isXaiTestnet = chainId === ChainId.XaiTestnet
   const isStylusTestnet = chainId === ChainId.StylusTestnet
 
-  const ethereumChainIds = [
-    ChainId.Mainnet,
-    ChainId.Rinkeby,
-    ChainId.Goerli,
-    ChainId.Sepolia,
+  const isEthereum =
```

Comment:
Bringing it back to the original state. Workaround was done to make it work with Orbit chains and cleaned up now.